common/check_new_dids

```diff
@@ -37,8 +46,8 @@ if __name__ == "__main__":
     try:
         registry = CollectorRegistry()
         session = get_session()
-        new_dids = 'select  /*+ index_FFS(dids DIDS_IS_NEW_IDX) */   count(1) from {schema}dids where is_new is not null'.format(schema=schema)
```

Comment:
I don’t have DB expertise, but I see no necessity to remove this hint for Oracle.

Reply:
Can I add hints in SQLAlchemy?

Reply:
Never mind, I found it. It doesn't seem to make a timing difference but it's included now